src/mon/MDSMonitor.cc

```diff
@@ -2385,16 +2390,14 @@ int MDSMonitor::legacy_filesystem_command(
   if (prefix == "mds set_max_mds") {
     // NOTE: deprecated by "fs set max_mds"
     int64_t maxmds;
-    if (!cmd_getval(g_ceph_context, cmdmap, "maxmds", maxmds) || maxmds < 0) {
+    if (!cmd_getval(g_ceph_context, cmdmap, "maxmds", maxmds) || maxmds <= 0) {
```

Comment:
No, you can just do cluster_down, fail the ranks, and then "fs rm".  I don't think there's any gain to formally deactivating rank 0 if you're about to destroy the FS.